TLM/TMPE.API/Notifier.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+
+namespace TrafficManager.API
+{
+    public class OnModifiedEventArgs
+    {
+        /// <summary>
+        /// node/segment (citizen/vehicle in future) that has been modified.
+        /// </summary>
+        public InstanceID InstanceID;
+
+        /// <summary>
+        /// Reserved: the sender that caused the rule to be modified.
+        /// </summary>
+        public object Sender;
+
+        /// <summary>
+        /// Reserved: The rule that has been modified. object type is decided by sender.
+        /// </summary>
+        public object Data;
+    }
+
+    public static class Notifier
+    {
+        public static event Action EventLevelLoaded;
+
+        // TODO [issue #967]: notify TTL Start/Stop events
+        public static event Action<OnModifiedEventArgs> EventModified;
+
+        public static void OnLevelLoaded() => EventLevelLoaded?.Invoke();
```

Comment:
Might be simplified to `EventLevelLoaded();` if you init field with `delegate { }` (no risk of nullReference), same for other events

Reply:
is it important ?

Reply:
Honestly it depends on how you like to trigger events. 
It's just single call, so it doesn't really matter here, but personally when I use events I usually set field to `delegate { }`
Summarizing:
```cs
public event Action EventLevelLoaded = delegate { };
// then...
EventLevelLoaded(); // don't care if someone is listening
```
or
```cs
public event Action EventLevelLoaded;
// then...
`EventLevelLoaded?.Invoke();`
// which translates to
if (EventLevelLoaded != null) 
   EventLevelLoaded(); // or EventLevelLoaded.Invoke(); if you like delegates
```
or very risky way
```cs
public event Action EventLevelLoaded;
// then...
EventLevelLoaded(); // will throw nullref exception if there are no listeners
```